### DISCUSSION TITLE
New default media types clash with how I want to name my document types

### DISCUSSION THREAD

**Discussion by JasonElkin** [umbraco/Umbraco-CMS#10483] [Features and ideas]
Does it make sense to create all of the new default media types a fresh install _without_ a starter kit?

I just started a new 8.14 project and went to create a new "Article" doctype but I couldn't because there's already a media type with that name/alias. 

It's not a _huge_ deal, I renamed the media type to "document" and carried on, but it strikes me that there are now 7 potential names for doctypes that, by default, are already used. 

Also, those media types are opinionated. e.g. "Article".

I'm not sure that "Article" actually make sense as a name for a media type? All of the file extensions explicitly reference "document" file types. I appreciate that there's potential for confusion with the concept of an Umbraco document, but is that terminology actually surfaced to editors?

![Screenshot 2021-06-16 145451](https://user-images.githubusercontent.com/7151793/122239732-174b1400-ceb9-11eb-85b3-2e2b70c7f7e6.png)

Anyway, that's just my opinion, and I don't think the answer is to pick the "right" one. Rather, I think Umbraco shouldn't have an opinion on this _unless_ i've chosen to install the starter kit.

**Comment by nul800sebastiaan** [comment]
Interesting, I actually didn't know that doctypes and mediatype couldn't have the same name, that's quite odd!

Yes, they should be there by default (IMO) but.. since this apparently has always been a problem, we've all just quietly worked around it by not making a doctype with the name "File"? 

Maybe we can find a way to not make them clash.. not sure how though.

**Reply by nul800sebastiaan** [reply]
Had a little discussion at HQ and of course I was too quick in my reply! 

Realized that it's the content type ALIAS that clashes, not the name. That's actually easy to fix, unfortunately not for the people who've already installed 8.14.0 but for 8.14.1 we'll prefix the alias with something like `umbMedia` (so in this case `umbMediaArticle`). So fresh installs from 8.14.1 forward will not run into this unfortunate clash, of course `article` is used very often!

**Reply by JasonElkin** [reply]
Haha, yes of course 🤦‍♂️ alias != name.

That works, thanks!

**Reply by nul800sebastiaan** [reply]
PR is here: https://github.com/umbraco/Umbraco-CMS/pull/10496

**Reply by marcemarc** [reply]
'Document' would still be a better name than 'Article' in this context though...

It's in the vernacular to say "Have you got that Word document ready" or we have "tons of PDF documents on our site"

Article can mean a generic 'item' eg an Item of clothing, or it can be 'a piece of writing, usually included with others in a publication' - but a PDF that is uploaded into a CMS might not always be a 'piece of writing'... but it is always a document!

eg an image isn't always a Portrait, and a sound file isn't always a concerto...

wondering if it's not called 'document' to avoid confusion with 'document type' - but should we actually be renaming 'document types'! (eg content types) - or is it a little too late in the day :-)

**Reply by JasonElkin** [reply]
💯 @marcemarc "document types" should totally be renamed "content types"! 

It makes a lot more sense, especially in the era of headless and complex content structures: nested content, blocks, compositions etc. are not "documents" but they _are_ "content".

That would actually match up with code too - content/content type terminology is used by the APIs.

**Reply by bjarnef** [reply]
Although it may be a good thing we have the new convention for these new media types, I think we have some inconsistent naming as we still have `File`, `Folder` and `Image` media type aliases.
https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Core/Constants-Conventions.cs#L106-L119

Furthermore the default member type alias is `Member`.
https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Core/Constants-Conventions.cs#L331-L334

These have used pascal casing for a long time and not consistent with new content types using camel case although I don't think the casing has impact on e.g. querying via UmbracoHelper or e.g. creating content / media based on the alias.

**Reply by nul800sebastiaan** [reply]
@bjarnef Yes and that won't change for a while to preserve backwards compatibility. We absolutely rely on these exact aliases for our internal code as well and while casing might not matter, we're not going to risk it in case it turns out that it actually does matter. :)
